.ci/gh_actions.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env python3
```

Comment:
If the plan is to migrate to actions, and remove travis later, perhaps it's worth having a commit to rename `travis.py` to `gh_actions.py` and then another commit to patch the changes such that the git blame history is preserved for the file we eventually keep.

Reply:
My concern with this was that until we convince ourselves that the GH approach does all what travis did we'd need to keep travis running as well.
But this can easily be reworked, travis (in opposition to github actions) can run cron jobs on any branch, so I can push the current `master` to a different branch and trigger from there if the gh needs additionnal love to do everything we want it to.